client/store/playbackManager.ts

```diff
@@ -140,6 +141,32 @@ export const getters: GetterTree<PlaybackManagerState, RootState> = {
         return stream.Type === 'Subtitle';
       });
     }
+  },
+  getCurrentTime: (state) => {
+    return state.currentTime;
+  },
+  getDuration: (state, _getters, _rootState, rootGetters) => {
+    if (state.currentItemIndex !== null) {
+      const currentItem = rootGetters['items/getItem'](
+        state.queue?.[state.currentItemIndex]
+      );
+
+      return ticksToMs(currentItem?.RunTimeTicks) / 1000;
+    }
+
+    return null;
+  },
+  getTimeLeft: (state, _getters, _rootState, rootGetters) => {
```

Comment:
Not sure about this either. It feels more like a computed property where you'd use it than something in the state itself.